src/main/java/io/minepkg/testutils/gui/ScreenDrawingTweak.java

```diff
@@ -28,7 +30,7 @@ public static void texturedRect(int x, int y, int width, int height, Identifier
     RenderSystem.enableBlend();
     //GlStateManager.disableTexture2D();
     RenderSystem.blendFuncSeparate(GlStateManager.SrcFactor.SRC_ALPHA, GlStateManager.DstFactor.ONE_MINUS_SRC_ALPHA, GlStateManager.SrcFactor.ONE, GlStateManager.DstFactor.ZERO);
-    buffer.begin(GL11.GL_QUADS, VertexFormats.POSITION_COLOR_TEXTURE); //I thought GL_QUADS was deprecated but okay, sure.
+    buffer.begin(VertexFormat.DrawMode.QUADS, VertexFormats.POSITION_COLOR_TEXTURE);
```

Comment:
"Deprecated" does not always mean "deprecated" in Minecraft, citing the Fabric Bot:
![deprecation](https://user-images.githubusercontent.com/8464472/125672274-fc6a03ec-fc40-4664-acc7-f658c620cd94.png)

Reply:
i don't think this was my code. this was some hack copy paste from libgui. i think